src/pages/api/updateFeedback.ts

```diff
@@ -38,9 +38,23 @@ const updateFeedbackHandler = async (
         return res.json({ message: 'This is a PUT-only endpoint.' });
     }
     try {
+        const { _id, ...content } = req.body;
+        // Need to make sure this is a valid ObjectID and not some random input.
+        const regHex = /^[0-9a-fA-F]{24}$/;
+        if (!regHex.test(_id)) {
+            res = res.status(400);
+            logRequestData(req.url, req.method, res.statusCode, req.headers);
+
+            return res.json({
+                error: {
+                    message:
+                        'Bad Request: "_id" value in request body must be a valid ObjectID.',
+                },
+            });
+        }
         const response = await axios.put(
-            process.env.REALM_API_URL + '/update_feedback',
-            req.body,
+            `${process.env.BACKEND_URL}/api/feedback/${_id}`,
```

Comment:
I don't think I know this functionality- when do we update feedback ?

Reply:
So we have different modal stages that each submit feedback. First the user clicks on the stars to rate the articles, which sends a POST request to `/api/createFeedback`, which enters that rating in our DB. Then they are shown a modal screen with checkboxes for why they like/dislike the article. when they hit "Next" on that modal, this endpoint is called, which updates that rating that was inserted in the POST request. Then they are shown a modal screen with text boxes for additional comments, which does the same thing by updating the document.

Reply:
LGTM. Read to approve once the conflicts are resolved.

Reply:
I would test kanopy to kanopy calls in staging after merging the code to staging.

Reply:
@harikakotipalli Just tested in a preview environment and it seems to be working well, but I will definitely check in staging as well.